crates/bin/pd/src/migrate/testnet74.rs

```diff
@@ -117,6 +148,9 @@ pub async fn migrate(
         // Write auction parameters
         write_auction_parameters(&mut delta).await?;
 
+        // Rewrite base liquidity indices as proto-encoded
+        rewrite_base_liquidity_indices(&mut delta).await?;
+
```

Comment:
When we circle back to addressing whether or not we want to enforce that all migrations are idempotent, we can sprinkle in some anyhow::Context usage on these calls. The tracing spans are already quite helpful in debugging what failed in a migration, but more context is never a bad thing IMO.